src/test/java/com/epam/tc/hw2/ex1/LoginPageTest.java

```diff
@@ -0,0 +1,97 @@
+        username.sendKeys("Roman");
+        webDriver.findElement(By.id("password"));
+        WebElement password = webDriver.findElement(By.id("password"));
+        password.sendKeys("Jdi1234");
+        WebElement loginButton = webDriver.findElement(By.id("login-button"));
+        loginButton.click();
+        webDriverWait.until(ExpectedConditions.urlToBe(EPAM_URL));
+        softly.assertThat(webDriver.getCurrentUrl()).isEqualTo(EPAM_URL);
+        // Check loggined
+        WebElement userNameLogged = webDriver.findElement(By.id("user-name"));
+        userNameLogged.isDisplayed();
+        webDriverWait.until(ExpectedConditions.textToBePresentInElement(userNameLogged, "ROMAN IOVLEV"));
+        softly.assertThat(userNameLogged.getText()).isEqualTo("ROMAN IOVLEV");
+        //Assert that there are 4 items on the header section are displayed and they have proper texts
+        List<WebElement> headerMenuWebElements = webDriver.findElements(By.xpath("//ul[contains(@class,'uui-navigation nav navbar-nav m-')]/li/a"));
+        headerMenuWebElements.get(0).isDisplayed();
+        headerMenuWebElements.get(1).isDisplayed();
+        headerMenuWebElements.get(2).isDisplayed();
+        headerMenuWebElements.get(3).isDisplayed();
+        softly.assertThat(headerMenuWebElements.get(0).getText()).isEqualTo("HOME");
+        softly.assertThat(headerMenuWebElements.get(1).getText()).isEqualTo("CONTACT FORM");
+        softly.assertThat(headerMenuWebElements.get(2).getText()).isEqualTo("SERVICE");
+        softly.assertThat(headerMenuWebElements.get(3).getText()).isEqualTo("METALS & COLORS");
+        //Assert that there are 4 images on the Index Page and they are displayed
+        List<WebElement> iconsWebElements = webDriver.findElements(By.className("benefit-icon"));
+        iconsWebElements.get(0).isDisplayed();
+        iconsWebElements.get(1).isDisplayed();
+        iconsWebElements.get(2).isDisplayed();
+        iconsWebElements.get(3).isDisplayed();
+        //Assert that there are 4 texts on the Index Page under icons and they have proper text
+        List<String> allTextsUnderIcon = new ArrayList<>();
+        List<WebElement> textUnderWebElements =
+            webDriver.findElements(By.className("benefit-txt"));
+        for (WebElement webElement : textUnderWebElements) {
+            allTextsUnderIcon.add(webElement.getText());
+        }
+        softly.assertThat(allTextsUnderIcon.get(0)).isEqualTo("To include good practices\n"
+            + "and ideas from successful\n"
+            + "EPAM project");
+        softly.assertThat(allTextsUnderIcon.get(1)).isEqualTo("To be flexible and\n"
+            + "customizable");
+        softly.assertThat(allTextsUnderIcon.get(2)).isEqualTo("To be multiplatform");
+        softly.assertThat(allTextsUnderIcon.get(3)).isEqualTo("Already have good base\n"
+            + "(about 20 internal and\n"
+            + "some external projects),\n"
+            + "wish to get more…");
```

Comment:
This could be simplified via using collections for the comparing elements.
please apply following fix for similar situations

```java
          List<String> expectedTextUnderIcons = List.of("To include good practices\n"
              + "and ideas from successful\n"
              + "EPAM project",
              "To be flexible and\ncustomizable",
               "Already have good base\n"
              + "(about 20 internal and\n"
              + "some external projects),\n"
              + "wish to get more…",
              "To be multiplatform");
          List<WebElement> textUnderWebElements =
              webDriver.findElements(By.className("benefit-txt"));
          List<String> allTextsUnderIcon = new ArrayList<>();
          for (WebElement webElement : textUnderWebElements) {
              allTextsUnderIcon.add(webElement.getText());
          }
          softly.assertThat(allTextsUnderIcon).containsInAnyOrder(expectedTextUnderIcons);
```